examples/openstack/main.tf

```diff
@@ -1,42 +1,61 @@
-provider metakube {
-  host = "set_it_up"
+terraform {
+  required_providers {
+    metakube = {
+      source  = "syseleven/metakube"
+      version = "0.1.1"
```

Comment:
I'm going to remove this line after the merge because when the version is not specified it pulls the latest!